source/Calamari.AzureAppService/Behaviors/TargetDiscoveryBehaviour.cs

```diff
@@ -155,6 +155,8 @@ public async Task Execute(RunningDeployment runningDeployment)
             catch (JsonException ex)
             {
                 Log.Warn($"Target discovery context from variable {contextVariableName} is in wrong format: {ex.Message}");
+                Log.Verbose("Raw JSON:");
+                Log.Verbose(json);
```

Comment:
Are there any security implications of logging the entire json string?

Reply:
Shoot this is not meant to be there! Good catch.

Reply:
Removed.